lib/components/common/Filters.tsx

```diff
@@ -1,113 +1,121 @@
-import { Popover, Box, Typography, Button, FormGroup } from '@mui/material';
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them